README.md

```diff
@@ -34,43 +64,70 @@ clouds:
       user_domain_name: ldap_users
     identity_api_version: 3
     cacert: /usr/share/pki/trust/anchors/SUSE_Trust_Root.crt.pem
+ansible:
+  use_hostnames: True
+  expand_hostvars: False
+  fail_on_errors: True
+```
+
+If you don't have the SUSE root certificate installed, check
+http://ca.suse.de/.
 
 You'll also need to pre-create some configuration in engcloud. It's
 convention here to use your username as part of the name of objects you create.
 
-Using the engcloud web interface or OpenStack CLI (i.e. "openstack
-keypair create..."), create a keypair
+Create a keypair on engcloud (using either the engcloud web interface or
+OpenStack CLI's `openstack keypair create`)
 for accessing the instances created. Remember the name of this
-keypair.
+keypair (named further `foctodoodle-key`)
 
-Using the engcloud web interface, create a network with a subnet. Take
-note of the network name you used. Then
-create a router with the 'floating' external network.  Then open the
+Create a network with a subnet. Take
+note of the network name you used (`foctodoodle-net`), and the subnet
+name (`foctodoodle-subnet`).
+
+Create a router with the `floating` external network.  Then open the
 router and add an interface to the network you created.
 
 Define the following environment variables prior to running the
 socok8s scripts:
 
+```
 # assuming you followed the example
-export OS_CLOUD=engcloud-me
+export OS_CLOUD=engcloud
 # the name of the keypair you created
 export KEYNAME=foctodoodle-key
 # your username plus whatever else you'd like, will be used for naming
 # objects you create in engcloud
 export PREFIX=foctodoodle
 # the name of the subnet you created
 export INTERNAL_SUBNET=foctodoodle-subnet
+```
 
 Prior to executing scripts, be aware that you may need to do some
 cleanup prior to retrying scripts or playbooks should they fail. In
 some steps a delete.sh script is provided to clean up any created
 resources. Reconfirming that you've done all the previous steps to set
 up now will save you some time later.
 
+## Deploying
+
 To begin a deployment from scratch, go to the root of your socok8s
-clone and run "./run.sh". This will run each of the seven top-level
+clone and run `./run.sh` or `./run.sh full-deploy`. This will run each of the seven top-level
```

Comment:
We might want to explain here that the default action is 'full-deploy' and what the other options are and how they are used. For the scope of what you're doing here, the first part (mentioning that 'full-deploy' is the default) is something I'd like to see addressed in this review, and the second part (explaining the other options) is something that could be done in a follow-up change. (I'm trying not to be one of those reviewers who would like you to fix every problem in the file in the current change along with giving everyone a free kitten and world peace :D)

Reply:
While I thought I was doing another PR for it, it was so small I did another commit to fix it, on top of this one. It still is the same context (improvements on the documentation)